frontend/src/app/office-hours/office-hours-page/office-hours-page.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
You can selete the `.spec` file here! Will not repeat this comment, but you can delete these everywhere - we do not have frontend testing yet.